feat(guide): Update Setup TorGuard for port forwarding Guide (#1974)
TRaSH-Guides/Guides: docs/Misc/How-to-setup-Torguard-for-port-forwarding.md

# Setup TorGuard for port forwarding

> These settings are a recommendation if you want to choose other settings be my guest.

Reviewer: ```suggestion
These are our supported and recommended settings, if you would like to choose others please keep that in mind.

If you want to support me, please use this [referral link](https://torguard.net/aff.php?aff=5575){:target="_blank" rel="noopener noreferrer"} and enter the following discount code:

Get 50% Off ALL Plans (Anonymous VPN, Anonymous VPN Pro, Anonymous VPN Premium)

```none
TRaSH-Guides-2024
```

[![torguard-FreeTrial-270x90](images/torguard/torguard-FreeTrial-728x90.gif)](https://torguard.net/aff.php?aff=5575){:target="_blank" rel="noopener noreferrer"}

> !!! bug "As of 13 March Torguard Settles Piracy Lawsuit and has agreed to use commercially reasonable efforts to block BitTorrent traffic on its servers in the US using firewall technology. :bangbang:<br><br>I Talked to several people and they are still able to use Torguard for Torrents, Perhaps because the connection is encrypted. And others just selected a server in another country.<br>- [Source Torguard](https://torguard.net/blog/why-torguard-is-blocking-bittorrent-on-us-servers/){:target="_blank" rel="noopener noreferrer"}.<br>- [Source Torrentfreak](https://torrentfreak.com/torguard-settles-piracy-lawsuit-and-agrees-to-block-torrent-traffic-on-u-s-servers-220314/){:target="_blank" rel="noopener noreferrer"}."

Reviewer 2: ```suggestion
!!! bug "As of 13 March 2022 Torguard Settles Piracy Lawsuit and has agreed to use commercially reasonable efforts to block BitTorrent traffic on its servers in the US using firewall technology. :bangbang:<br><br>I Talked to several people and they are still able to use Torguard for Torrents, Perhaps because the connection is encrypted. And others just selected a server in another country.<br>- [Source Torguard](https://torguard.net/blog/why-torguard-is-blocking-bittorrent-on-us-servers/){:target="_blank" rel="noopener noreferrer"}.<br>- [Source Torrentfreak](https://torrentfreak.com/torguard-settles-piracy-lawsuit-and-agrees-to-block-torrent-traffic-on-u-s-servers-220314/){:target="_blank" rel="noopener noreferrer"}."
```
specify year

> !!! warning "If the US server doesn't work for you use another country"

Reviewer: ```suggestion
!!! warning "If U.S. servers are not working for you, please try another country"

Reviewer 2: ![image](https://github.com/TRaSH-Guides/Guides/assets/55419169/c032c844-d69c-42b5-acb1-c14ec2991807)
https://academicguides.waldenu.edu/writingcenter/apa/other/abbreviations

Reviewer: updated.

For the best speed it's recommended to choose the one nearest to your location.

> !!! tip "Sometimes you get better speeds if you use one of your neighboring countries."

Reviewer: ```suggestion
!!! tip "Sometimes you will have better speeds if you choose a neighboring country."

!!! warning "If the US server doesn't work for you use another country"

> Let's use for this example: `Netherlands Amsterdam` and copy/paste the Hostname `nl.torguard.org`.

Reviewer: ```suggestion
For this example, let's use: `Netherlands Amsterdam` and copy/paste the hostname `nl.torguard.org`.

![!Torguard Global VPN Network](images/torguard/globalvpnnetwork-list.png)

> !!! tip "Also check if your chosen server supports Wireguard"

Reviewer: ```suggestion
!!! tip "Also check if your selected server supports Wireguard"

### Get your VPN IP

Open a cmd window or something equal and type:

```bash
    ping nl.torguard.org
```

You will then get the IP of the server.
Remember that for later use.

![!ping](images/torguard/cmd-ping.png)

---

## How to setup Port forwarding

> From your Client Area go to your [My Services](https://torguard.net/clientarea.php?action=products){:target="\_blank" rel="noopener noreferrer"}.

Reviewer: ```suggestion
From your `Client Area` dashboard, go to [`My Services`](https://torguard.net/clientarea.php?action=products){:target="\_blank" rel="noopener noreferrer"}.

    ![!Port Forward Request](images/torguard/req_port_fwd.png)

### Port Forward Request

![!Request New Port Forward WireGuard](images/torguard/request-new-pfw-wireguard.png)

1. That's the `IP` we found earlier.
1. Select `UDP`. (:bangbang: **KEEP THIS ON UDP FOR WireGuard** :bangbang:)

> 1. Select a `Port/Auth` and select `WireGuard`.

Reviewer: ```suggestion
1. Select `Port/Auth` and select `WireGuard`.

1. Select the `Protocol` `TCP`.

> 1. I suggest to use a high `Port` number 10.000+ or a game port you don't use, don't use the default torrent ports 6881- 6889

Reviewer 2: ```suggestion
1. I suggest to use a high `Port` number 10000+ or a game port you don't use, don't use the default torrent ports 6881- 6889
```
no comma, this is a port number not a numeric number - do not use a separator

Reviewer: yep. this.

    (This is also the port you will use in your torrent client).

> 1. Click on the `+` sign and do the same with the `UDP` `Protocol`.

Reviewer: ```suggestion
1. Click on the `+` sign, and do the same with the `UDP` `Protocol`.

1. Then click on `Submit Request`.

!!! warning ":bangbang: DON'T CHANGE OPTION 2 `UDP` to `TCP` :bangbang:"

> If everything went according to plan you will see something like this.

Reviewer: ```suggestion
If everything succeeds, you will see the following.

![status](images/torguard/status.png)

You will also receive a e-mail with the ports you forwarded.

> !!! Note "Normally it doesn't take more then a few seconds/minutes to get the ports approved if it takes longer or you get `PENDING` I suggest to ping the server again (or choose another sever) and you often get a new IP and retry it again with the new IP."

Reviewer: ```suggestion
!!! Note "Normally it takes only a short while for the ports to be approved, if it takes longer or you get `PENDING` I suggest pinging the server again (or choose another server) to attempt to get a new IP. You can then retry the process with the new IP."

---

## How to create the config file

> From your Client Area go to your [Config Generator](https://torguard.net/tgconf.php?action=vpn-openvpnconfig){:target="\_blank" rel="noopener noreferrer"}.

Reviewer: ```suggestion
From your `Client Area` dashboard, go to your [`Config Generator`](https://torguard.net/tgconf.php?action=vpn-openvpnconfig){:target="\_blank" rel="noopener noreferrer"}.

### Config Generator

![!WireGuard Config Generator](images/torguard/config-generator-wireguard.png)

1. Choose `WireGuard`.
1. Choose from the dropdown box the `IP` we got earlier and used for the port forwarding.
1. Add your `VPN Username`.
1. Only Change this if you know what you're doing :bangbang:
1. [Optional] Choose you preferred DNS Server
1. Only Change this if you know what you're doing :bangbang:
1. Click on `Generate Config`

> Now you will download a `.conf` file with a random number.

Reviewer: ```suggestion
This will generate a download of a `.conf` file with a random number.

> Rename the file from `*.conf` to `wg0.conf` and copy it to your wireguard folder of your VPN torrent client.

Reviewer: ```suggestion
Rename the .config file you just downloaded to `wg0.conf` and copy it to the wireguard folder of your VPN torrent client.
